### DISCUSSION TITLE
Tranapose operation in onnx runtime using c++

### DISCUSSION THREAD

**Discussion by ashray21** [microsoft/onnxruntime#15196] [Other Q&A]
I have successfully inferences the yolov8 output from the onnx runtime. The size of the ort::value vector is 1X84X8400. I need to transpose this vector into 1X8400X84 to post-process the output. How can I do this?

**Comment by justinchuby** [comment]
I would modify the ONNX model to add a transpose node.

**Reply by ashray21** [reply]
Hey justinchuby can you give me some reference or any repo to how to do that.

**Reply by pranavsharma** [reply]
This is really a question for the ONNX community which supports a list of APIs to manipulate the ONNX graph. See https://onnx.ai/onnx/api/index.html.

**Reply by ashray21** [reply]
Thanks, my issue got solved.
